homeassistant/components/lcn/helpers.py

```diff
@@ -425,6 +425,16 @@ async def async_update_config_entry(
     hass.config_entries.async_update_entry(config_entry, data=new_data)
 
 
+def get_device_config(
+    address: AddressType, config_entry: ConfigEntry
+) -> ConfigType | None:
+    """Return the device configuration for given unique_device_id from ConfigEntry."""
```

Comment:
The docstring does not seem to match the implementation?

Reply:
Changed